prettier/main.ts

```diff
@@ -2,8 +2,10 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 // This script formats the given source files. If the files are omitted, it
 // formats the all files in the repository.
-const { args, readAll, lstat, exit, readFile, writeFile } = Deno;
-import { xrun } from "./util.ts";
+const { args, exit, readFile, writeFile } = Deno;
+import { FileInfo } from "deno";
```

Comment:
This should be `type FileInfo = Deno.FileInfo;` because 'deno' builtin module is going to be removed soon.  https://github.com/denoland/deno/pull/1895

Reply:
Fixed!